### PR TITLE
Parsed and calculated

### PR DESCRIPTION
**Pull request by bohdan-tron**

### CODE REVIEW

**Comment by bohdan-tron** [comment]
[DEMO LINK](https://bohdan-tron.github.io/js_get_data_DOM/)

File: src/scripts/main.js [hunk 1]
```diff
@@ -1,3 +1,19 @@
 'use strict';
 
-// write your code here
+let sum = 0;
+let count = 0;
```
Comment (SanyaBratashchuk) [inline src/scripts/main.js:4]: why do you need a count variable? use length of the array

File: src/scripts/main.js [hunk 2]
```diff
@@ -1,3 +1,19 @@
...
+
+document.querySelectorAll('.population').forEach((item) => {
+  sum += parseInt(item.innerHTML.split(',').join(''));
+  count++;
+});
+
+function numberWithCommas(x) {
```
Comment (SanyaBratashchuk) [inline src/scripts/main.js:11]: Use verbs for function names

File: src/scripts/main.js [hunk 3]
```diff
@@ -1,3 +1,19 @@
...
+  return x.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',');
```
Comment (SanyaBratashchuk) [inline src/scripts/main.js:12]: use number.toLocalString() with correct argument to format numbers or read about Intl

**Pull request closed without merging**